plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetadata.java

```diff
@@ -775,7 +776,8 @@ public Stream<TableColumnsMetadata> streamTableColumns(ConnectorSession session,
                 }
                 return Stream.empty();
             }
-        });
+        })
+                .iterator();
```

Comment:
Previous line

Reply:
see my answer to https://github.com/trinodb/trino/pull/12687#discussion_r889754654